conf/cmi/helfi_kymp_content.project_search.yml

```diff
@@ -1 +1 @@
-project_search_path: ''
+project_search_path: /node/39
```

Comment:
Should this config change be here or not?

Reply:
Probably not. Good catch!!

Reply:
Fixed.